lib/qmp.mli

```diff
@@ -89,6 +89,8 @@ type message =
   | Event of event
 (** an individual message sent or received to or from qemu *)
 
+val _JSONParsing : string
```

Comment:
for code using the library that wants to compare the string returned in Error.cls during a json parsing error.